### PR TITLE
fix for heroku ci

### PR DESCRIPTION
**Pull request by kennethreitz**

### CODE REVIEW

**Comment by kennethreitz** [comment]
@jmorrell good idea

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Python Buildpack Changelog
 
+## 103
+
+Bug fixes.
+
+- Fix for Pipenv.
+- Fix for Heroku CI.
```
Comment (edmorley) [inline CHANGELOG.md]: Looking at https://github.com/heroku/heroku-buildpack-python/compare/v102...e621ff4, there's also:
- Improve handling of `WEB_CONCURRENCY` when using multiple buildpacks.
- Adjust environment variables set during the build to more closely match those in the dyno environment (`DYNO` is now available, `STACK` is not).
- Restore the build cache prior to running `bin/pre_compile`.

:-)

Comment (kennethreitz) [inline CHANGELOG.md]: @edmorley thanks :)

Comment (kennethreitz) [inline CHANGELOG.md]: @edmorley i'd love to hop on a hangout with you sometime soon if you're available! send me an email kenneth@heroku.com

**Pull request merged**